### PR TITLE
feat(providers): added providers for baidu, sina and wechat (#1)

### PR DESCRIPTION
**Pull request by zacksleo**
fixed #11

### CODE REVIEW

File: tests/Mremi/UrlShortener/Tests/Provider/Wechat/WechatProviderTest.php [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+
+    /**
+     * Tests the expand method throws exception if Wechat returns a string.
+     *
+     * @expectedException        \Mremi\UrlShortener\Exception\InvalidApiResponseException
+     * @expectedExceptionMessage Wechat does not support expand url yet.
+     */
+    public function testExpandThrowsExceptionIfApiResponseIsString()
+    {
+        //$this->mockClient($this->getMockResponseAsString());
```
Comment (mremi) [inline tests/Mremi/UrlShortener/Tests/Provider/Wechat/WechatProviderTest.php:133]: Can you explain please?

Comment (zacksleo) [reply tests/Mremi/UrlShortener/Tests/Provider/Wechat/WechatProviderTest.php:133]: see https://github.com/mremi/UrlShortener/pull/11#discussion_r219786491

**Comment by mremi** [comment]
Can you update the CHANGELOG.md please?

**Comment by zacksleo** [comment]
image

You can choose the option called Squash and merge, then the PR will be merged into one commit.

**Comment by mremi** [comment]
Thx @zacksleo

**Comment by zacksleo** [comment]
You are welcome, can you make a new release ?

**Comment by mremi** [comment]
Done https://github.com/mremi/UrlShortener/releases/tag/v2.2.0

**Pull request merged**